guide/C/ch_invest.xml

```diff
@@ -1813,6 +1813,840 @@ Income
+
+          <listitem>
+            <para>Run a Trial Balance report to ensure total debits balance to
+              total credits.</para>
+          </listitem>
+
+        </orderedlist>
+
+      </sect3>
+
+      <sect3 id="invest-sellManual">
+        <title>Manual Lot Creation and Linking</title>
+
+        <para>Before using this feature, ensure you have read
+          <xref linkend="invest-sellConsiderations" />.
+        </para>
+
+        <para>This functionality allows the manual linking of specfic buy and
+          sell transactions. It may be used in the case where a user wishes to
+          use a different cost method than the automatic linking method (FIFO).
+          Effectively, if one wishes the cost basis and capital gains in the
+          Advanced Portfolio Report to be consistent with the capital gains
+          transactions created by scrubbing, manual lot creation only needs to
+          be used when using LIFO or <quote>sale of designated lots</quote>
+          (the same thing for securities as far as US personal tax law is
+          concerned). This is because the scrub function can automatically do
+          FIFO linking so there is no need to do it manually and scrubbing
+          cannot be used for average costing.
+        </para>
+
+        <para>See <xref linkend="invest-sellManualExample" />.
+        </para>
+      </sect3>
+
+      <sect3 id="invest-sellAuto">
+        <title>Automatic Creation of Capital Gain Or Loss Transactions</title>
+        <note><para>Do <emphasis>NOT</emphasis> do this unless you are using
+          FIFO or LIFO to cost sales. See
+          <xref linkend="invest-sellConsiderations" /></para>
+        </note>
+
+        <para><application>&app;</application> can automatically calculate and
+          create security sale capital gain/loss transactions. Lots are used to
+          link buy transaction splits with sell transaction splits so the
+          correct cost of the securities sold can be determined.
+          <application>&app;</application> will use any existing lots, and
+          create new lots for any buy transaction splits not already linked to a
+          lot. Buy and sell transaction splits are linked to lots using FIFO
+          method.
+        </para>
+
+        <para>See <xref linkend="invest-sellAutoExample" />.
+        </para>
+      </sect3>
+
+      <sect3 id="invest-sellChgCapGainsAcct">
+        <title>Change Orphaned Gains-CCC to Gain/Loss Account</title>
+
+        <para>
+          The capital gain/loss transaction(s) created by scrubbing uses an
+          automatically created generic <emphasis>Orphaned Gains-CCC</emphasis>
+          account (where CCC is the security currency) because <application>
+          &app;</application> doesn't know which capital gain or loss account
+          should be used. After scrubbing, the user should change it to the
+          required income (or expense) gain or loss account.
+        </para>
```

Comment:
Thanks for picking that up. How about:
          After scrubbing, the user should edit the <emphasis>
          Orphaned Gains-CCC</emphasis> transaction split to re-assign the
          income account to a more meaningful income (or expense) gain or loss
          account (e.g. Income:Long Term Capital Gain:IBM).